wandb/sdk/internal/job_builder.py

```diff
@@ -296,14 +299,23 @@ def build(self) -> Optional[Artifact]:
         if not os.path.exists(
             os.path.join(self._settings.files_dir, REQUIREMENTS_FNAME)
         ):
+            wandb.termwarn(
+                "No requirements.txt found, not creating job artifact. See https://docs.wandb.ai/guides/launch/create-job"
+            )
             return None
         metadata = self._handle_metadata_file()
         if metadata is None:
+            wandb.termwarn(
+                "No metadata file found, not creating job artifact. Ensure api settings files_dir exists with read and write access."
```

Comment:
I dont think a user could do much with this warning. "Ensure read and write access to run files dir: {self._settings.files_dir}, control this via the WANDB_DIR env var" or something?

Reply:
Maybe even link to the env var docs...